apps/betterangels/package.json

```diff
@@ -81,7 +81,8 @@
     "@teovilla/react-native-web-maps": "*",
     "react-native-paper": "*",
     "dotenv": "*",
-    "@react-native-picker/picker": "*"
+    "@react-native-picker/picker": "*",
+    "react-native-image-zoom-viewer": "*"
```

Comment:
Any concerns with this lib being archived and not updated in over 4 years?
Looks like it's still getting more weekly downloads then the alternatives, so perhaps it's fine.

Reply:
other one was 3 years ago. but downloads were 2 times less.
react-native-image-viewing